tests/playbooks/tests_bond_options.yml

```diff
@@ -9,201 +9,51 @@
-                  arp_interval: 60
-                  arp_ip_target: 192.0.2.128
-                  arp_validate: none
-                  primary: "{{ dhcp_interface1 }}"
-                ip:
-                  route_metric4: 65535
-              # add an ethernet to the bond
-              - name: "{{ port1_profile }}"
-                state: up
-                type: ethernet
-                interface_name: "{{ dhcp_interface1 }}"
-                controller: "{{ controller_profile }}"
-              # add a second ethernet to the bond
-              - name: "{{ port2_profile }}"
-                state: up
-                type: ethernet
-                interface_name: "{{ dhcp_interface2 }}"
-                controller: "{{ controller_profile }}"
-        - name: "** TEST check bond settings"
-          command: cat
-            /sys/class/net/{{ controller_device }}/bonding/'{{ item.key }}'
-          register: result
-          until: "'{{ item.value }}' in result.stdout"
-          loop:
-            - {key: 'mode', value: 'active-backup'}
-            - {key: 'arp_interval', value: '60'}
-            - {key: 'arp_ip_target', value: '192.0.2.128'}
-            - {key: 'arp_validate', value: 'none'}
-            - {key: 'primary', value: '{{ dhcp_interface1 }}'}
-          changed_when: false
-
-        - name: "** TEST check IPv4"
-          command: ip -4 a s {{ controller_device }}
-          register: result
-          until: "'192.0.2' in result.stdout"
-          retries: 20
-          delay: 2
-          changed_when: false
-        - name: "** TEST check IPv6"
-          command: ip -6 a s {{ controller_device }}
-          register: result
-          until: "'2001' in result.stdout"
-          retries: 20
-          delay: 2
-          changed_when: false
-
-      always:
-        - name: Clean up the test devices and the connection profiles
-          tags:
-            - "tests::cleanup"
-          block:
-            - name: Import network role
-              import_role:
-                name: linux-system-roles.network
-              vars:
-                network_connections:
-                  - name: "{{ port2_profile }}"
-                    persistent_state: absent
-                    state: down
-                  - name: "{{ port1_profile }}"
-                    persistent_state: absent
-                    state: down
-                  - name: "{{ controller_profile }}"
-                    persistent_state: absent
-                    state: down
-              failed_when: false
-            - name: Delete the device '{{ controller_device }}'
-              command: ip link del {{ controller_device }}
-              failed_when: false
-              changed_when: false
-            - name: Import the task 'remove_test_interfaces_with_dhcp.yml'
-              import_tasks: tasks/remove_test_interfaces_with_dhcp.yml
+            lsr_description: I can reconfigure the bond options
```

Comment:
I wrote the description in user story format before I knew about acceptance criteria. Probably, it makes more sense to write this as an acceptance criteria.

Reply:
I wrote the `lsr_description` as acceptance criteria.